docs/data/data-grid/aggregation/AggregationRowGrouping.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import {
+  DataGridPremium,
+  GridColDef,
+  useGridApiRef,
+  useKeepGroupedColumnsHidden,
+} from '@mui/x-data-grid-premium';
+import { useMovieData } from '@mui/x-data-grid-generator';
+
+const COLUMNS: GridColDef[] = [
+  { field: 'title', headerName: 'Title', width: 200, groupable: false },
+  {
+    field: 'company',
+    headerName: 'Company',
+    width: 200,
+  },
+  {
+    field: 'gross',
+    headerName: 'Gross',
+    type: 'number',
+    width: 150,
+    groupable: false,
+    valueFormatter: ({ value }) => {
+      if (!value || typeof value !== 'number') {
+        return value;
+      }
+      return `${value.toLocaleString()}$`;
+    },
+  },
+];
+
+export default function AggregationRowGrouping() {
+  const data = useMovieData();
+  const apiRef = useGridApiRef();
+
+  const initialState = useKeepGroupedColumnsHidden({
+    apiRef,
+    initialState: {
+      rowGrouping: {
+        model: ['company'],
+      },
+      aggregation: {
+        model: {
+          gross: {
+            // Aggregation displayed on the footers
+            footer: 'sum',
```

Comment:
I already saw this bug in other parts but it was not as perceptible as in your video. Here's one taken from https://mui.com/x/react-data-grid/row-grouping/#full-example

Observe the small jumps when I scroll up

https://user-images.githubusercontent.com/42154031/169629913-fabbe316-5e81-4b19-bc43-b621e678c96e.mp4

The same happens with the `actions` column type but we don't have a demo with many rows to see it. I don't know why it happens. We already use `HTMLElement.focus({ preventScroll: true })` but it seems that it still tries to scroll to make the element visible. One thing I noticed is that it only happens if there's a cell focused so it might be coming from the effect inside the cell:

https://github.com/mui/mui-x/blob/e5b548f6744f6d000436879839067055205e07db/packages/grid/x-data-grid/src/components/cell/GridCell.tsx#L179

Reply:
I also got the same bug... no idea where it come from :/

Reply:
I opened #5006 since the bug is not only in this PR.